<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QPushButton" name="newTask">
      <property name="text">
       <string>New Task</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTreeWidget" name="treeWidget">
      <attribute name="headerDefaultSectionSize">
       <number>110</number>
      </attribute>
      <column>
       <property name="text">
        <string>id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>header</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>worker</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>status</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>New Item</string>
       </property>
       <property name="background">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Categories</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Workers</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
